### ISSUE TITLE
Remove $group_ids parameter from Og::invalidateCache()

### ISSUE THREAD

**Issue by pfrenssen** [Gizra/og#218]
According to the documentation, `Og::invalidateCache()` accepts a `$group_ids` parameter which should contain the IDs of groups that should have their cache invalidated.

This can never work since the information about the entity types of the groups are not passed. For the moment this parameter is only used to pass it on to a `hook_og_invalidate_cache($group_ids)`.

```
  /**
   * Invalidate cache.
   *
   * @param array $group_ids
   *   Array with group IDs that their cache should be invalidated.
   */
  public static function invalidateCache(array $group_ids = array()) {
  }
```

We have a number of options to solve this:

1. Remove the parameter from the method and the hook.
1. Pass the full group entities instead of the IDs.
1. Pass a two-dimensional array `[$entity_type => [$entity_id]]`

**Comment by amitaibu** [comment]
I think (1) would be the simplest.

**Comment by pfrenssen** [comment]
Great choice :) That's pretty easy to do, can you maybe tag this as a novice issue?

**Comment by amitaibu** [comment]
Sure. You now have commit access to the repo, so you can label as-well. Congrats 😄

**Comment by pfrenssen** [comment]
Thanks!

**Comment by alvar0hurtad0** [comment]
Hi,

I've done a PR about this issue:

https://github.com/Gizra/og/pull/234

Thanks.

**Comment by zerolab** [comment]
This was fixed in #234, so can be closed
